ross/bearing_seal_element.py

```diff
@@ -1736,17 +1739,17 @@ def load(file_name=""):
         return bearing_elements
 
     def dof_mapping(self):
-        return dict(x_0=0, y_0=1, z_0=2)
+        return dict(u_0=0, v_0=1, w_0=2)
```

Comment:
Hello @raphaeltimbo , we kept the standard used in the definition of the element matrices. The differences are cosmetic, so if there is the need, we can change back to (x,y,z) without altering the code function.

Reply:
I agree these don't alter the code function. I am more worried about the documentation for the user. There are parts of our documentation using `x`, `y`... So I think it would be better to follow this convention so that someone reading the docs don't get confused.

Reply:
Corrected here (https://github.com/ross-rotordynamics/ross/pull/531/commits/1c3dc0992e4d5833d3e3f5a9fc3f3b693e39cec4)